frontend/components/redeem/steps/PersonalContributionStep.tsx

```diff
@@ -51,7 +52,7 @@ const PersonalContributionStep = ({
 
         <Stack component="div" alignItems="center" spacing={1}>
           <Typography variant="h2" align="center">
-            Do you want to add a personal contribution too?
+            {couponSponsorship ? 'Do you want to add a personal contribution too?' : 'How much would you like to give?'}
           </Typography>
 
           <Typography variant="body2" align="center">
```

Comment:
Could you help me add `placeholder="0"` prop at line 68? This will solve the problem of having a blank field when a user doesn't want to donate